core/src/main/java/org/apache/accumulo/core/client/mapreduce/AccumuloOutputFormat.java

```diff
@@ -563,8 +563,6 @@ public void close(TaskAttemptContext attempt) throws IOException, InterruptedExc
 
   @Override
   public void checkOutputSpecs(JobContext job) throws IOException {
-    if (!isConnectorInfoSet(job))
-      throw new IOException("Connector info has not been set.");
```

Comment:
The message could be improved, but this is not an unnecessary check. The convenience methods we provide to set the configuration on the job do not need to be called. It's entirely possible somebody will try to use our input format or output format without properly configuring it.